web/src/components/Plugin/PluginPage.tsx

```diff
@@ -56,31 +58,63 @@ const PluginPage: React.FC<Props> = ({
     getList().then(setPlugin);
   }, []);
 
+  // NOTE: This function has side effect because it mutates the original schema data
+  const injectDisableProperty = (schema: Record<string, any>) => {
```

Comment:
only a little confused about  this func.
since manager-api already inject `disable` property, why we need do again in fe?

Reply:
We just did what manager-api does 😂 

The manager-api doesn't respond that property with the schema, so we have to trick that property in FE.

Reply:
got it